emr-serverless-sample/README.md

```diff
@@ -0,0 +1,112 @@
+# Running EMR Serverless Jobs with Java
```

Comment:
nit: since this entire repo is about samples, maybe rename the directory to `emr-serverless` or `emr-serverless-spark`?